libyul/optimiser/Suite.cpp

```diff
@@ -139,7 +140,13 @@ void OptimiserSuite::run(
 			_optimizeStackAllocation,
 			stackCompressorMaxIterations
 		);
-	suite.runSequence("fDnTOc g", ast);
+
+	// Run the user-supplied clean up sequence
+	suite.runSequence(_optimisationCleanupSequence, ast);
+	// Hard-coded FunctionGrouper step used to bring the AST into a canonical form required by the StackCompressor
```

Comment:
Without "is" this reads as past tense at first.
```suggestion
	// Hard-coded FunctionGrouper step is used to bring the AST into a canonical form required by the StackCompressor
```